apis/tools/ci/README.md

```diff
@@ -0,0 +1,52 @@
+## PyServer CI setup
+
+We use pre-commit along with ruff to lint and format our code.
+
+This combination gives us auto-formatting on save, lint warnings and prevents us from uploading non-compliant code when committing changes.
+
+The configurations can be found at the `.pre-commit-config.yaml` and `.ruff.toml` files at the root of the repository.
+
+To start using them, just install the tools following the steps below.
+
+### Ruff
+
+Install ruff:
+
+```console
+pip install ruff --break-system-packages
+```
```

Comment:
The `--break-system-packages` option is required when using Python 3.11 or greater. Since the Python version in the Wazuh code is 3.10, we should make the differentiation or mention this.